### ISSUE TITLE
Do not insert moments-only source-fits into the database.

### ISSUE THREAD

**Issue by timstaley** [transientskp/pyse#72]
The basic logic here is that if we cannot fit a Gaussian, probably there is something wrong with modelling these pixels as a typical source and we should not accept the fit at all. There's also the issue that different fitting methods provide different errors, corrections etc. and it's nice to have a homogenous dataset, if that only means dropping a handful of (already poor) fits.

The long version is at https://redmine.astron.nl/lofar_issuetracker/issues/6041

**Comment by HannoSpreeuw** [comment]
> I am puzzled by [commit 68a2f49](https://github.com/transientskp/tkp/commit/68a2f49326f8e60d6221cd2eae16438be064a068):

I will revert the part of that commit that affects `sourcefinder/extract.py`, since any negative peak spectral brightness from a moment analysis is impossible. 
It cannot occur when `detectionthresholdmap` is "sane", i.e. all its values are positive.

The only circumstance I can think of.....some values in the noise map can be negative, i.e. the map from the interpolated rms node values (always positive) can be negative if a higher order interpolation scheme is used. 
The detection threshold (=positive float) times the rms noise map can in that case be negative for some part of `detectionthresholdmap`. 
But this cannot happen if `INTERPOLATE_ORDER = 1`. 
That had already been set as a default value prior to this issue, I guess starting from [commit dabc8a0, from April 15, 2014](https://github.com/transientskp/tkp/commit/dabc8a0663a4a981682ce6ba3fcf6aa4354def10).

What the setting of infinite error bars should probably be replaced with  is raising a ValueError:
```
if peak < 0:
    raise ValueError((f"Peak from moments = {a:.2e}, something is possibly "
                       "wrong with detectionthresholdmap, since a negative "
                       "peak from moments analysis should not occur."))

```

**Comment by HannoSpreeuw** [comment]
Fixed by commits [4b94f16](https://github.com/transientskp/pyse/commit/4b94f166cb2e3a8bdbc71576858db8750140faf2), [e81bbe5](https://github.com/transientskp/pyse/commit/e81bbe54b440208994115105bb42accd9def0236) and [9257eee](https://github.com/transientskp/pyse/commit/9257eeed338150c3ba7000e36e21933fe5ad1ed4).

But merge with `master` pending.
